homeassistant/components/bsblan/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/bsblan",
   "iot_class": "local_polling",
   "loggers": ["bsblan"],
-  "requirements": ["python-bsblan==0.5.9"]
+  "requirements": ["python-bsblan==0.5.10"]
```

Comment:
The fix was reverted in the library dev branch after 0.5.10 was released. Was there a problem with the fix?

https://github.com/liudger/python-bsblan/commit/66117f44ee714cede20704b15c1db5695891ab7b

Reply:
Yes that is a mistake indeed. I have a look and fix the issue. (I really thought I tested the code in dev environment and could set the thermostat to max)
Thanks for the check 👍

Reply:
Which was the mistake, the fix in 0.5.10 or the revert of the fix?

Reply:
the revert that overruled the pull request with the fix. I unintentionally pushed the commit to main branch and did a revert. With the pull request with the fix the revert also reverted that pull request. Not sure why? As the revert was before the pull request.

Reply:
Ok, thanks!